pylint/checkers/classes/class_checker.py

```diff
@@ -1331,7 +1346,11 @@ def _check_property_with_parameters(self, node):
         ):
             self.add_message("property-with-parameters", node=node)
 
-    def _check_invalid_overridden_method(self, function_node, parent_function_node):
+    def _check_invalid_overridden_method(
+        self,
+        function_node: nodes.AsyncFunctionDef | nodes.FunctionDef,
```

Comment:
Let's just use `FunctionDef` for all of these

Reply:
Done